### PR TITLE
Add a minimum threshold for STREAM_ALL_AUDIO

### PR DESCRIPTION
**Pull request by glebm**

### CODE REVIEW

File: CMake/platforms/rg99.cmake [hunk 1]
```diff
@@ -25,6 +25,7 @@ set(DEVILUTIONX_PALETTE_TRANSPARENCY_BLACK_16_LUT OFF)
 
 # Must stream all the audio due to RAM constraints.
 set(STREAM_ALL_AUDIO ON)
+set(STREAM_ALL_AUDIO_MIN_FILE_SIZE 4096)
```
[the remark this answers is not in the document]
Comment (glebm) [reply CMake/platforms/rg99.cmake:28]: I'm not sure how that would work?
We want to respect `sfx_MISC` by default.

`STREAM_ALL_AUDIO` = stream even `sfx_MISC` sounds
`STREAM_ALL_AUDIO_MIN_FILE_SIZE` = only stream files above a certain size, regardless of `sfx_MISC` / `sfx_STREAM`

[the remark this answers is not in the document]
Comment (glebm) [reply CMake/platforms/rg99.cmake:28]: With your other suggestion (applied):

`STREAM_ALL_AUDIO_MIN_FILE_SIZE` = only stream `sfx_MISC` sounds if they're above a certain size

[the remark this answers is not in the document]
Comment (glebm) [reply CMake/platforms/rg99.cmake:28]: Ah, I see, yeah perhaps we could do that in a subsequent cleaup

[the remark this answers is not in the document]
Comment (glebm) [reply CMake/platforms/rg99.cmake:28]: Thinking further about it, these are actually 2 different settings, I think they should be kept as 2 separate variables.
With just `STREAM_ALL_AUDIO_MIN_FILE_SIZE`, to disable all-audio streaming you'd have to set it to a very high value which is a bit confusing.

[the remark this answers is not in the document]
Comment (AJenbo) [reply CMake/platforms/rg99.cmake:28]: you would leave it undefined. nothing makes use of all streaming after this change

[the remark this answers is not in the document]
Comment (glebm) [reply CMake/platforms/rg99.cmake]: Done

**Review by AJenbo — APPROVED** [review]
Thanks, looks very clean now :)

**Pull request merged**